crates/matrix-sdk-sqlite/src/crypto_store.rs

```diff
@@ -1377,6 +1409,42 @@ mod encrypted_tests {
         );
     }
 
+    #[async_test]
+    async fn cache_cleared_after_device_update() {
```

Comment:
Is there some test we can add to `cryptostore_integration_tests` for this? Then it would cover indexeddb, which I think is not covered so far in this PR.

Reply:
I don't think we can add this to `cryptostore_integration_tests` because `memorystore` works differently, and always keeps the sessions in cache (the cache can't be cleared). (But yes, I should add a test for indexeddb -- once I figure out how to run the indexeddb tests locally)

Reply:
Here is now I ran the tests locally here:

```
cd crates/matrix-sdk-indexeddb/
WASM_BINDGEN_TEST_TIMEOUT=2400 CARGO_TARGET_DIR=/tmp/andyb-rust-target wasm-pack test --firefox --headless -- --no-default-features --features e2e-encryption
```

Reply:
I tried that, but it gave me an error saying:

```
Try find `webdriver.json` for configure browser's capabilities:
Not found
driver status: signal: 9 (SIGKILL)
```

I wrote a test anyways and it seems to pass CI, so I'll ignore it for now, and try to figure it out later if needed.